app/src/main/java/com/spendster/presentation/homeScreen/Dashboard/DashboardModel.java

```diff
@@ -0,0 +1,9 @@
+package com.spendster.presentation.homeScreen.Dashboard;
+
+import com.spendster.data.entity.Expense;
+
+import java.util.List;
+
+public interface DashboardModel {
+    List<Expense> getExpenses();
```

Comment:
@SimonMaha Maybe it would be better to wrap response into observable? As data will come from the server.